comp/core/tagger/impl/tagger.go

```diff
@@ -6,7 +6,7 @@
 // The Tagger is the central source of truth for client-side entity tagging.
 // It subscribes to workloadmeta to get updates for all the entity kinds
 // (containers, kubernetes pods, kubernetes nodes, etc.) and extracts the tags for each of them.
-// Tags are then stored in memory (by the TagStore) and can be queried by the tagger.Tag()
+// Tags are then stored in memory (by the tagStore) and can be queried by the tagger.Tag()
```

Comment:
Are we able to delete this wrapper now?

Is there something that prevents us from deleting it or it's just that you prefer to do it in a future PR to avoid introducing more changes on this one?

Reply:
I wanted to do it in a follow-up PR to limit the scope of this one as it's already a bit big 👍